packages/apps/viewer-sample-react/src/components/home/AuthClientHome.tsx

```diff
@@ -79,6 +79,23 @@ export const AuthClientHome = () => {
           appInsightsKey={process.env.REACT_APP_APPLICATION_INSIGHTS_KEY}
           backend={backend}
           theme={ColorTheme.Dark}
+          defaultUiConfig={{
+            contentManipulationTools: {
+              hideDefaultHorizontalItems: true,
```

Comment:
Do we need more granularity than this?  Or is the idea that you disable the entire group and then add back things that you want?

Reply:
yes. That was the idea. My thinking is to that the general goal is to use the UI Providers in UI2.0 to build the toolbars that each application desires. This basically would give them a clean slate. We can expand it into individual buttons if we see a large base that desires that, but it's just as easy (and preferable) to add the core tools to their own UI providers.